README.md

```diff
@@ -36,19 +36,13 @@ write
 
 ## Installation
 
-FastDom is CommonJS and AMD compatible, you can install it in one of the following ways:
+In your package.json
 
-``` sh
-$ npm install fastdom
-```
-``` sh
-$ bower install fastdom
 ```
-``` sh
-$ component install wilsonpage/fastdom
+    "fastdom": "github:Financial-Times/fastdom#v2.1.1"
```

Comment:
I'm not really sure we should have this project recommend potential package consumers use a fixed version. v2.1.1. is just the current latest tag. 

I think it this could potentially be more confusing in the future not less? 

I personally think it would be fine to just remove the `Installation` steps from this README - so there is no potentially misleading info OR do something like [`ft-icons`](https://github.com/Financial-Times/ft-app-icons ) and have something along the lines of `Pull in a new version of this dependency into your app repo, edit package.json and either npm update or npm install`

Thoughts?